deploy/yamls/zfs-driver.yaml

```diff
@@ -587,7 +587,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: openebs-zfs-plugin
-          image: quay.io/openebs/zfs-driver:ci
+          image: quay.io/openebs/zfs-driver:0.6.0
```

Comment:
will this impact travis builds?

Reply:
no, only testing will be imacted as it will not use the currently build ci image, instead, it will use quay.io/openebs/zfs-driver:0.6.0 image.